astroid/scoped_nodes.py

```diff
@@ -1797,6 +1797,17 @@ def __init__(self, name=None, doc=None, lineno=None,
         if parent is not None:
             parent.frame().set_local(name, self)
 
+        for name, node in self.implicit_locals():
+            self.add_local_node(node, name)
+
+    def implicit_locals(self):
+        """Get implicitly defined locals locals"""
+        locals_ = (('__module__', self.special_attributes.py__module__),)
```

Comment:
Can we have this a static, with the exception of the `if` statement? e.g no method for it.

Reply:
Ah, I see, nevermind!